src/CliLogger.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of composer/xdebug-handler.
+ *
+ * (c) Composer <https://github.com/composer>
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+namespace Composer\XdebugHandler;
+
+use Psr\Log\AbstractLogger;
+
+/**
+ * Writes XdebugHandler status messages to stdout in a CLI process.
+ *
+ * @author John Stevenson <john-stevenson@blueyonder.co.uk>
+ */
+class CliLogger extends AbstractLogger
```

Comment:
Imo this library should not provide an implementation, only the plugin hook, or it still does not solve the SRP concern. We really don't need to reinvent that wheel, there are enough logger implementations on Composer. I don't need every library I use to provide one bundled. It's just another thing that is part of the public interface, subject to semver, could have bugs (and releases for the fixes). For example, what if some developer tries to pass in `$context` and is confused why it doesn't get logged. The next developer asks for the default output to be STDERR not STDOUT. We shouldn't have to worry about these things.

Reply:
I cheekily included this because there was (and will be) a logger implementation in the library (for tests), but in retrospect I tend to agree with what you have said.

Reply:
I guess you can include monolog as a dev requirement

Reply:
Yeah, or use PHPUnits mocking functionality. But if it's in the tests it's not that important (as long as we exclude the test folder from the Composer package so it's not part of the public interface).